script/makeinstaller.sh

```diff
@@ -43,14 +43,23 @@ do
     touch "${build_dir_plugins}/$i.py"
 done
 
+echo "Creating images"
```

Comment:
```suggestion
echo "Creating images" 1>&2
```

Send the output to `stderr`, better for logging :)